### ISSUE TITLE
TVer support broken

### ISSUE THREAD

**Issue by vivian-ng** [yt-dlp/yt-dlp#3261]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.03.08.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Japan

### Description

It used to be possible to download from TVer (tver.jp) when its URL was in this format.
https://tver.jp/feature/f0101104

Today, TVer revamped its website, and the URL is now like this.
https://tver.jp/episodes/epgqep1rqb

This throws an "Unsupported URL" error when using yt-dlp.

### Verbose log

```shell
[debug] Command-line config: ['-vU', 'https://tver.jp/episodes/epgqep1rqb']
[debug] User config "/home/teck/.config/yt-dlp/config": ['-o', '~/Downloads/%(title)s.%(ext)s', '-f', 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/mp4', '--trim-filenames', '80', '--no-mtime', '--all-subs', '--embed-subs']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2022.03.08.1 [c0c2c57d3] (zip)
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.13.0-39-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4, rtmpdump 2.4
[debug] Optional libraries: Cryptodome, secretstorage, sqlite
[debug] Proxy map: {}
Latest version: 2022.03.08.1, Current version: 2022.03.08.1
yt-dlp is up to date (2022.03.08.1)
[debug] [generic] Extracting URL: https://tver.jp/episodes/epgqep1rqb
[generic] epgqep1rqb: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] epgqep1rqb: Downloading webpage
[generic] epgqep1rqb: Extracting information
[debug] Looking for video embeds
ERROR: Unsupported URL: https://tver.jp/episodes/epgqep1rqb
Traceback (most recent call last):
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1389, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1459, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 617, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/generic.py", line 3985, in _real_extract
    raise UnsupportedError(url)
yt_dlp.utils.UnsupportedError: Unsupported URL: https://tver.jp/episodes/epgqep1rqb
```

**Comment by someziggyman** [comment]
I can confirm the issue, for some reason link processing is sent to Generic extractor.
Here's another test link:
https://tver.jp/episodes/ep07dpaso7

**Comment by vivian-ng** [comment]
This [PR](https://github.com/yt-dlp/yt-dlp/pull/3268) currently works. As a temporary workaround until it gets into release, you can run the script directly using `yt-dlp.sh` by cloning the repo, then making sure that you have the revised `tver.py` file from the PR.

**Comment by hatienl0i261299** [comment]
> This [PR](https://github.com/yt-dlp/yt-dlp/pull/3268) currently works. As a temporary workaround until it gets into release, you can run the script directly using `yt-dlp.sh` by cloning the repo, then making sure that you have the revised `tver.py` file from the PR.

Actually, i think you dont need to wait the new release. You can install by `pip install git+https://github.com/yt-dlp/yt-dlp` to install with the latest code from repo,

**Comment by vivian-ng** [comment]
Fixed in the latest release on 2022.04.08
:grinning:
